extension/tests/event-page-data-webtest.js

```diff
@@ -27,7 +27,7 @@ describe('EventPage', function() {
       ]);
       assert.isOk(result);
       assert.isOk(
-          result.hasOwnProperty('urla'),
+        result.hasOwnProperty('urla'),
```

Comment:
I think the original indent is at least closer to what I would write - here and below. I assume these changes were not intentional, were they?

Reply:
I'll revert those changes.  My editor likes 2sp continuation indents.

At some point the project will want to use clang-format or prettier or something.  That day is not today :)